app/calculator.js

```diff
@@ -0,0 +1,56 @@
+/**
+ *
+ * A simple class which performs the summarize and multiply actions
+ * @class Calculator
+ */
+
+class Calculator {
+  /**
+   * Creates an instance of Calculator.
+   * @param {*} numbers is the incoming value for the calculations
+   * @memberof Calculator
+   */
+  constructor(numbers) {
+    this.number = numbers;
```

Comment:
What is the use of this argument?
Why wouldn't you just leave empty constructor?
`constructor() {}`